### PR TITLE
!roll the dice

### PR DESCRIPTION
**Pull request by richarm4**

### CODE REVIEW

File: modules/general.rb [hunk 1]
```diff
@@ -91,6 +91,13 @@ module GeneralCommands
       command(:lug, aliases: %i[plug]) do
         'https://discord.gg/3Jfq6aXy5B 🔌'
       end
+
+      command(:roll) do |event, n|
+        if n.to_i <= 0
+          "You need to put in an integer that is equal to or greater than 1."
+        else
```
Comment (detjensrobert) [inline modules/general.rb:98]: lets make this simpler and have the bot react with :question: if its not an int or not >1

File: modules/general.rb [hunk 2]
```diff
@@ -91,6 +91,13 @@ module GeneralCommands
...
+          "You rolled: #{rand(1..n)}"
```
Comment (detjensrobert) [inline modules/general.rb:99]:
```suggestion
          "You rolled: #{rand(1..n.to_i)}"
```

**Review by detjensrobert — CHANGES_REQUESTED** [review]

File: modules/general.rb (hunk 1 above)
Comment (richarm4) [reply modules/general.rb]: Fixed

Comment (detjensrobert) [reply modules/general.rb]: I mean like `event.react "?"` _a la_ `!role`

Comment (richarm4) [reply modules/general.rb]: Ah, will re-fix XD

**Review by detjensrobert — APPROVED** [review]
rolled a 19

**Pull request merged**